src/coreclr/src/pal/src/exception/machexception.cpp

```diff
@@ -1422,25 +1422,21 @@ ActivationHandler(CONTEXT* context)
 {
     if (g_activationFunction != NULL)
     {
+#if defined(HOST_OSX) && defined(HOST_ARM64)
+        auto jitWriteEnableHolder = PAL_JITWriteEnable(false);
```

Comment:
A nit - I would prefer this to be in the activation function in the VM instead.

Reply:
I shouldn't have reverted this line....  Its not needed here... I'll delete it...